www/_posts/2021-02-13-inappbrowser-release-5.0.0.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+author:
+    name: Niklas Merz
+    url: https://twitter.com/niklasmaerz
+title:  "Cordova Plugin InAppBrowser 5.0.0 Released!"
+categories: announcements
+tags: news releases plugins
+---
+
+We are happy to announce that we have just released an update to `cordova-plugin-inappbrowser`! 
+
+* [cordova-plugin-inappbrowser@5.0.0](https://www.npmjs.com/package/cordova-plugin-inappbrowser)
+
+## Release Highlights
+
+The most notable improvements in this major release are:
+
+* The InAppBrowser and main webview now share web resources again (like cookies, sessions etc.).
+* You can now set `InAppBrowserStatusBarStyle` to 'darkcontent'.
+* The Android part got some code cleanup.
```

Comment:
I think we should somewhere mention that this is breaking and now required API level XYZ

Reply:
Good point. I added a sentence with version restrictions from: https://github.com/apache/cordova-plugin-inappbrowser/pull/823/files